test/ReaderIO.ts

```diff
@@ -0,0 +1,102 @@
+import * as assert from 'assert'
+import { pipe } from 'fp-ts/lib/pipeable'
+import * as R from 'fp-ts/lib/Reader'
+import * as IO from 'fp-ts/lib/IO'
+import * as _ from '../src/ReaderIO'
+
+describe('ReaderIO', () => {
```

Comment:
Same as https://github.com/gcanti/fp-ts-contrib/pull/90#discussion_r806656801 and https://github.com/gcanti/fp-ts-contrib/pull/91#discussion_r827149693. 😃